scripts/create_gitlab_snapshot.py

```diff
@@ -0,0 +1,60 @@
+import datetime
+import logging
+from more_itertools import (
+    one,
+)
+
+from azul import (
+    config,
+    require,
+)
+from azul.deployment import (
+    aws,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+
+from azul.terraform import (
+    _adjust_name_tag,
+    _tags,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def create_snapshot():
+    require(config.terraform_component == 'gitlab',
+            'Please _select the GitLab environment for the targeted deployment')
+    filters_ = [
+        {
+            'Name': 'key-name',
+            'Values': ['azul-gitlab']
+        }
+    ]
+    instance = one([
+        one(reservation['Instances'])
+        for reservation in aws.ec2.describe_instances(Filters=filters_)['Reservations']
+    ])
+    assert instance['State']['Name'] == 'stopped', instance['State']
+    resource_tags = [
+        {'Key': k, 'Value': v}
+        for k, v in _adjust_name_tag('aws_gitlab',
+                                     _tags('gitlab', name='azul_gitlab')).items()
```

Comment:
Not sure what this buys us. `_adjust_name_tag` and `_tags` are taylored for use in TF config. I mean you artificially hard-code `aws_gitlab` simply to trigger the renaming of `Name` to `name`. There is a reason `_adjust_name_tag` is protected. Isn't it easier to just hard-code the tags here?

Reply:
I wanted there to be one less place where there was hard-coded tags so they can stay in-band with any changes that are made to the tags. 

Its hard-coded now.